velox/exec/tests/DriverTest.cpp

```diff
@@ -818,6 +826,294 @@ TEST_F(DriverTest, driverCreationThrow) {
       "Can only create 1 'throw driver'.");
 }
 
+DEBUG_ONLY_TEST_F(DriverTest, driverSuspensionRaceWithTaskPause) {
+  std::vector<RowVectorPtr> batches;
+  for (int i = 0; i < 4; ++i) {
+    batches.push_back(
+        makeRowVector({"c0"}, {makeFlatVector<int32_t>({1, 2, 3})}));
+  }
+
+  struct {
+    int numDrivers;
+    bool enterSuspensionAfterPauseStarted;
+    StopReason expectedEnterSuspensionStopReason;
```

Comment:
this and 'expectedLeaveSuspensionStopReason' seem to be always kNone. Is this expected? Do we need these variables?

Reply:
It is expected and we can remove it.